Makefile.am

```diff
@@ -184,8 +184,14 @@ lib_LTLIBRARIES          += motr/libmotr.la
 motr_libmotr_la_CPPFLAGS  = -DM0_TARGET='libmotr' $(AM_CPPFLAGS)
 motr_libmotr_la_LDFLAGS   = -version-info @LT_VERSION@ -pthread $(AM_LDFLAGS)
 motr_libmotr_la_LIBADD    = @MATH_LIBS@ @PTHREAD_LIBS@ @AIO_LIBS@ @RT_LIBS@ \
-                            @YAML_LIBS@ @PROFILER_LIBS@ @UUID_LIBS@ @GALOIS_LIBS@ \
-                            @DL_LIBS@ @CASSANDRA_LIBS@ @UV_LIBS@ @ISAL_LIBS@
+                            @YAML_LIBS@ @PROFILER_LIBS@ @UUID_LIBS@ \
+                            @DL_LIBS@ @CASSANDRA_LIBS@ @UV_LIBS@
+
+if HAVE_ISAL
```

Comment:
Please also fix the "cortx-motr.spec.in" file:
```
cortx-motr.spec.in:106:BuildRequires:  isa-l
cortx-motr.spec.in:134:Requires:       isa-l
```

These two options should be based on HAVE_ISAL

Reply:
We need to spend some time to handle this as HAVE_ISAL is not visible in "cortx-motr.spec.in" file. I think we can address this in **[EOS-20623](https://jts.seagate.com/browse/EOS-20623) - Avoid Galois code compilation when using Intel ISA**.

Reply:
Thanks Hua for the patch. I have pushed the changes.